### PR TITLE
FTP-765: Configure Jetpack-Compose

### PR DESCRIPTION
**Pull request by OmarShehe**
**Ticket:** [FTP-765](https://endios.atlassian.net/browse/FTP-765 "Link to JIRA")

**Purpose of this Pull Request:**
- Updated Kotlin plugin version; `ComposeCompiler` must be compatible with a specific `Kotlin` version; I update it to the latest version.
- Updated play_services_maps version
- Added jetpack compose dependencies
- Added accompanist system ui controller dependency

[FTP-765]: https://endios.atlassian.net/browse/FTP-765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: version.gradle [hunk 1]
```diff
@@ -74,12 +76,19 @@ version.okhttp = "4.9.1"
 // permission
 version.permission = "4.8.0"
 // play-services
-version.play_services_maps = "18.0.0"
+version.play_services_maps = "18.1.0"
```
Comment (nicolaiendios) [inline version.gradle:79]: Are there any breaking changes that come with this version?

File: version.gradle [hunk 2]
```diff
@@ -29,7 +29,9 @@ version.coil = "1.1.0"
 // core
 version.android_gradle_plugin = '7.2.0'
 version.kotlin_coroutines = "1.6.0"
-version.kotlin = "1.6.0"
+// When updating please update compose_kotlin_compiler too
+// see https://developer.android.com/jetpack/androidx/releases/compose-kotlin
+version.kotlin = "1.8.21"
```
Comment (nicolaiendios) [inline version.gradle:34]: Are there any breaking changes that come with updating to this version?

File: version.gradle (hunk 1 above)
Comment (OmarShehe) [reply version.gradle:79]: @nicolaiendios From what I read from the [Google release notes](https://developers.google.com/maps/documentation/android-sdk/release-notes#August_29_2022), no, there are no breaking changes

File: version.gradle (hunk 2 above)
Comment (OmarShehe) [reply version.gradle:34]: Also, since it's a big jump from version `1.6.0` to `1.8.21`, we may face some lint warnings

**Review by JimArabajoglou — DISMISSED** [review]

**Pull request merged**